tests/build/build_templates.go

```diff
@@ -155,37 +158,25 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				var pipelineRun *v1beta1.PipelineRun
 
 				BeforeAll(func() {
-					// Create an Service account and a token associating it with the service account
-					resultSA := "tekton-results-tests"
-					_, err := kubeadminClient.CommonController.CreateServiceAccount(resultSA, testNamespace, nil)
+					// create the proxyplugin for tekton-results
+					_, err = kubeadminClient.CommonController.CreateProxyPlugin("tekton-results", "toolchain-host-operator", "tekton-results", "tekton-results")
 					Expect(err).NotTo(HaveOccurred())
-					_, err = kubeadminClient.CommonController.CreateRoleBinding("tekton-results-tests", testNamespace, "ServiceAccount", resultSA, "ClusterRole", "tekton-results-readonly", "rbac.authorization.k8s.io")
-					Expect(err).NotTo(HaveOccurred())
-
-					resultSecret := &v1.Secret{
-						ObjectMeta: metav1.ObjectMeta{
-							Name:        "tekton-results-tests",
-							Annotations: map[string]string{"kubernetes.io/service-account.name": resultSA},
-						},
-						Type: v1.SecretTypeServiceAccountToken,
-					}
 
-					_, err = kubeadminClient.CommonController.CreateSecret(testNamespace, resultSecret)
-					Expect(err).ToNot(HaveOccurred())
-					err = kubeadminClient.CommonController.LinkSecretToServiceAccount(testNamespace, resultSecret.Name, resultSA, false)
-					Expect(err).ToNot(HaveOccurred())
-
-					resultSecret, err = kubeadminClient.CommonController.GetSecret(testNamespace, resultSecret.Name)
-					Expect(err).ToNot(HaveOccurred())
-					token := resultSecret.Data["token"]
-
-					// Retrieve Result REST API url
-					resultRoute, err := kubeadminClient.CommonController.GetOpenshiftRoute("tekton-results", "tekton-results")
+					// Retrieve registration-proxy REST API url for tekton-results data
+					proxyRoute, err := kubeadminClient.CommonController.GetOpenshiftRoute("api", "toolchain-host-operator")
 					Expect(err).NotTo(HaveOccurred())
-					resultUrl := fmt.Sprintf("https://%s", resultRoute.Spec.Host)
-					resultClient = pipeline.NewClient(resultUrl, string(token))
 
-					pipelineRun, err = kubeadminClient.HasController.GetComponentPipelineRun(componentNames[i], applicationName, testNamespace, "")
+					regProxyUrl := fmt.Sprintf("https://%s/plugins/tekton-results", proxyRoute.Spec.Host)
+					resultClient = pipeline.NewClient(regProxyUrl, f.UserAccessToken)
+
+					err = wait.Poll(pipelineCreatedRetryInterval, pipelineCreatedTimeout, func() (done bool, err error) {
+						pipelineRun, err = f.AsKubeDeveloper.HasController.GetComponentPipelineRun(componentNames[i], applicationName, testNamespace, "")
+						if err != nil {
```

Comment:
again some logging of the error so it does not get  masked/lost if this poll times out